src/OpenSage.Game/UserSettingsFiles/FontFallbackSettings.json

```diff
@@ -0,0 +1,26 @@
+﻿{
```

Comment:
I'd probably put this file to Content/Translation or Content/Fonts

Reply:
All right, will change it to Content/Fonts